microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -66,6 +67,7 @@ const (
 	redis_vector_db_yaml             = "/redis-vector-db.yaml"
 	retriever_yaml                   = "/retriever.yaml"
 	reranking_yaml                   = "/reranking.yaml"
+	doc_sum_yaml                     = "/backend-service.yaml"
```

Comment:
this is the mega service for docsum, gmc should not use it

Reply:
Yeah, remove it.

Reply:
Hi @irisdingbj, this is not mega service, it's a new microservice for docsum example, we need to add it in. Please refer to https://github.com/opea-project/GenAIComps/tree/main/comps/llms/summarization/tgi for detail.